content/reference/user/artifact_conf/index.md

```diff
@@ -11,25 +11,22 @@ weight: 10050
 
 To access the Artifacts section, click on **Artifacts** in the navbar.
```

Comment:
in the SideNav, not the navbar

Reply:
Isn't the side bar called the "navigation bar" elsewhere in the docs? (Either way, neither "SideNav" nor "navbar" seem appropriate terminology for the user facing docs.)

Reply:
Sidenav is the one on the left side, and navbar at the top side. Indeed, the naming can get a bit confusing, and may need a change

Reply:
I checked, we do seem to call it navigation bar everywhere else in the docs. I'm going to use that; if it's not correct then it should be changed globally.